patches/chrome-browser-BUILD.gn.patch

```diff
@@ -10,3 +10,11 @@ index 59773d17657448a508a8cea31ca3a1f72f3c5abc..689f8850f8a0c6d879fab9f04d01b278
  
    if (!is_chromeos_ash) {
      sources += [
+@@ -6658,6 +6659,7 @@ static_library("browser") {
+       "enterprise/connectors/connectors_manager.h",
+     ]
+     deps += [ "//components/enterprise/connectors/core" ]
++    deps += [ "//chrome/browser/enterprise/connectors/analysis:features" ]
```

Comment:
why in the world is this ending up in here? Is this an upstream patch?

Reply:
This was temporary while waiting for the upstream fix, which was just merged yesterday (https://chromium-review.googlesource.com/c/chromium/src/+/5828216), but hasn't been uplifted to cr129 yet. I will update it to match the upstream fix.

Reply:
Updated in https://github.com/brave/brave-core/pull/24966/commits/0b60e7e45965c031f5dc82a3f8596e8aa346fe1b